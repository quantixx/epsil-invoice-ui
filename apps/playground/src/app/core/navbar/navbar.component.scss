//

:host {
  display: block;

  // As we have a 56px fixed top navbar
  height: 56px;
}

.navbar {
  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12),
    0 2px 4px -1px rgba(0, 0, 0, 0.2);
}
