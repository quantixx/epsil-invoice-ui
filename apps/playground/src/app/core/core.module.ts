//

import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { RouterModule } from '@angular/router';
import { NavbarComponent } from './navbar/navbar.component';
import { PlaygroundAppRootComponent } from './playground-app-root/playground-app-root.component';

//

@NgModule({
  declarations: [
    PlaygroundAppRootComponent,
    PlaygroundAppRootComponent,
    NavbarComponent
  ],
  exports: [PlaygroundAppRootComponent],
  imports: [CommonModule, RouterModule]
})
export class CoreModule {}

export { PlaygroundAppRootComponent };
