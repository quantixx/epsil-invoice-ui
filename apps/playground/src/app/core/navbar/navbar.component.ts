//

import { Component } from '@angular/core';

//

@Component({
  selector: 'epsil-playground-navbar',
  styleUrls: ['./navbar.component.scss'],
  templateUrl: './navbar.component.html'
})
export class NavbarComponent {}
