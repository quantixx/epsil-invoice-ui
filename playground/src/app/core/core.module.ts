import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { RouterModule } from '@angular/router';
import { PlaygroundAppRootComponent } from './playground-app-root/playground-app-root.component';

@NgModule({
  imports: [CommonModule, RouterModule],
  declarations: [PlaygroundAppRootComponent, PlaygroundAppRootComponent],
  exports: [PlaygroundAppRootComponent]
})
export class CoreModule {}

export { PlaygroundAppRootComponent };
