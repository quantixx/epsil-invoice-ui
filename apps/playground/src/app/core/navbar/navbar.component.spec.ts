//

import { TestBed } from '@angular/core/testing';
import { RouterTestingModule } from '@angular/router/testing';
import { NavbarComponent } from './navbar.component';

//

describe('Integration::NavbarComponent', () => {
  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [NavbarComponent],
      imports: [RouterTestingModule]
    }).compileComponents();
  });

  it('should create', () => {
    const fixture = TestBed.createComponent(NavbarComponent);
    const component = fixture.componentInstance;

    fixture.detectChanges();

    expect(component).toBeTruthy();
    expect(fixture).toMatchSnapshot();
  });
});
