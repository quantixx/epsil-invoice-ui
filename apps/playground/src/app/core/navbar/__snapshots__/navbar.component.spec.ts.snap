// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Integration::NavbarComponent should create 1`] = `
<epsil-playground-navbar>
  
  
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light"
  >
    
    
    <a
      class="navbar-brand"
      href="/"
      ng-reflect-router-link="/"
      routerlink="/"
    >
      Playground
    </a>
    
    
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      class="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      
      
      <span
        class="navbar-toggler-icon"
      />
      
    
    </button>
    
  
    
    <div
      class="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      
      
      <ul
        class="navbar-nav mr-auto"
      >
        
        
        <li
          class="nav-item"
        >
          
          
          <a
            class="nav-link"
            href="/"
            ng-reflect-router-link="/"
            ng-reflect-router-link-active="active"
            ng-reflect-router-link-active-options="[object Object]"
            routerlink="/"
            routerlinkactive="active"
          >
            
            Home
          
          </a>
          
        
        </li>
        
        
        <li
          class="nav-item"
        >
          
          
          <a
            class="nav-link"
            href="/invoices"
            ng-reflect-router-link="/invoices"
            ng-reflect-router-link-active="active"
            routerlink="/invoices"
            routerlinkactive="active"
          >
            
            Invoices
          
          </a>
          
        
        </li>
        
      
      </ul>
      
  
    
    </div>
    
  
  </nav>
  
  
  
</epsil-playground-navbar>
`;
